atlasdb-backup/src/test/java/com/palantir/atlasdb/backup/transaction/Transactions1TableInteractionTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.backup.transaction;
+
+import static com.palantir.atlasdb.backup.transaction.Transactions1TableInteraction.COLUMN_NAME_BB;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Statement;
+import com.datastax.driver.core.TableMetadata;
+import com.datastax.driver.core.policies.RetryPolicy;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Range;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.cassandra.CassandraConstants;
+import com.palantir.atlasdb.transaction.encoding.V1EncodingStrategy;
+import com.palantir.atlasdb.transaction.impl.TransactionConstants;
+import java.nio.ByteBuffer;
+import org.apache.commons.codec.binary.Hex;
+import org.junit.Test;
+
+public class Transactions1TableInteractionTest {
```

Comment:
We can extract `AbstractTransactionsTableInteractionTests` to capture common behaviour here (again, my copypasta senses are tingling)

Reply:
Yeah, we try to avoid abstract test classes these days, though I was tempted

Reply:
agree it's a little janky - if you manage to pull out one common class via composition, then you can just test the composed class.